packages/backend/media-optimiser/src/media-optimiser.ts

```diff
@@ -23,6 +23,7 @@ export const optimiseMedia = async ({
     "jpeg"
   ],
   sizes = [
+    375,
     600,
     960,
     1200
```

Comment:
I feel we should bump `1200` to `1280` as `1280` is a real-world (realistic) value